### DISCUSSION TITLE
Package for version 4 on https://deb.jesec.io/

### DISCUSSION THREAD

**Discussion by sin0light** [jesec/flood#764] [General]
Hi !

Hope you're doing well. I saw the new version 4 and I am hyped for it ! However I would like to know if you intend to publish these new versions on the https://deb.jesec.io/ package repository ?

Sinolight

**Comment by trim21** [comment]
I want to, but I can't publish new version on deb.jesec.io, only jesec can do that.

**Reply by sin0light** [reply]
@jesec do you intend to publish on the repo ?

**Comment by sin0light** [comment]
Hi @jesec,

Do you intend to publish these new versions on the https://deb.jesec.io/ package repository ?

Sinolight
